samigo/samigo-pack/src/bundle/samigo-noti-prefs.properties

```diff
@@ -2,6 +2,6 @@ prefs_title = Tests & Quizzes
 # Do not remove prefs_description or prefs_title_override, check SAK-21078 for risk and code for reason
 prefs_description =
 prefs_title_override =
-prefs_opt3 = Send me an email confirmation each time I submit an assessment
-prefs_opt2 = Send me one email per day confirming all of my submissions
-prefs_opt1 = Do not send me any email confirmations
+prefs_opt3 = Seperate Best\u00e4tigung f\u00fcr jede Einreichung per Email empfangen
```

Comment:
Can you revert these changes? You're changing the default locale.

Reply:
done